_data/library-infrastructure-packages.yml

```diff
@@ -249,3 +249,18 @@
     dead-letter queues, and IP-based access controls. Create SNS topics with configurable IAM and delivery policies.
     Create Kinesis streams with configurable or auto-calculated shard and retention settings.
 
+- name: "Kubernetes / EKS"
+  tags:
+    - AWS
+  icons:
+    - title: Terraform
+      image: icon-terraform.png
+    - title: Python
+      image: icon-python.png
```

Comment:
Not a blocker for this PR, but it occurs to me that someday we should have k8s logo or tag so we can mark which of our packages support k8s.

Reply:
Oh I was actually thinking that it belongs on the cloud, but that makes sense too!